src/main/java/com/onarandombox/MultiverseCore/commands/ImportCommand.java

```diff
@@ -46,33 +43,17 @@ public ImportCommand(MultiverseCore plugin) {
         this.worldManager = this.plugin.getMVWorldManager();
     }
 
-    private String getPotentialWorlds() {
-        File worldFolder = this.plugin.getServer().getWorldContainer();
-        if (worldFolder == null) {
-            return "";
-        }
-        File[] files = worldFolder.listFiles();
-        String worldList = "";
-        Collection<MultiverseWorld> worlds = this.worldManager.getMVWorlds();
-        List<String> worldStrings = new ArrayList<String>();
-        for (MultiverseWorld world : worlds) {
-            worldStrings.add(world.getName());
-        }
-        for (String world : this.worldManager.getUnloadedWorlds()) {
-            worldStrings.add(world);
-        }
+    private String getPotentialWorldStrings() {
+        final Collection<String> potentialWorlds = this.worldManager.getPotentialWorlds();
+        StringBuilder worldList = new StringBuilder();
         ChatColor currColor = ChatColor.WHITE;
-        for (File file : files) {
-            if (file.isDirectory() && WorldNameChecker.isValidWorldFolder(file) && !worldStrings.contains(file.getName())) {
-                worldList += currColor + file.getName() + " ";
-                if (currColor == ChatColor.WHITE) {
-                    currColor = ChatColor.YELLOW;
-                } else {
-                    currColor = ChatColor.WHITE;
-                }
-            }
+
+        for (String world : potentialWorlds) {
+            worldList.append(currColor).append(world).append(' ');
+            currColor = currColor == ChatColor.WHITE ? ChatColor.YELLOW : ChatColor.WHITE;
         }
```

Comment:
We should probably have a utility method that does this since we do it in several places. Something that takes like vararg colors and a collection of items maybe?